server/src/main/java/org/candlepin/model/ContentAccessCertificate.java

```diff
@@ -69,6 +72,18 @@ public void setConsumer(Consumer consumer) {
         this.consumer = consumer;
     }
 
+    public String getContent() {
+        return content;
+    }
+
+    public void setContent(String content) {
+        this.content = content;
+    }
+
+    public String getFullCert() {
```

Comment:
This feels wrong to me. Rather than have a special method that builds the "full" cert, the default "getCert" behavior should be the full thing, and the specific components should be fetchable individually.

Anything using the cert generically should not have to fumble with specifics of the SCA cert. It should only be when the individual components of the SCA cert are of interest that we start working with explicit classes and custom methods.

Reply:
Rollbacked to the wrapping solution.